dunwoody_disaster/views/MapScreen.py

```diff
@@ -10,8 +10,11 @@
 
 
 class MapScreen(QWidget):
+    battle_start = Signal(object)
```

Comment:
Confused on what this is.

Reply:
battle_start is a signal call which is used in the keypress function where when a user clicks on a map level, it emits a signal to main where it will pass relevant data to main so it knows what room it is, what enemy is in there and etc.

Reply:
What is `object`?